indico/modules/events/persons/__init__.py

```diff
@@ -32,3 +32,17 @@
 def _sidemenu_items(sender, event, **kwargs):
     if event.can_manage(session.user):
         return SideMenuItem('persons', _('Roles'), url_for('persons.person_list', event), section='reports')
+
+
+@signals.get_placeholders.connect_via('event-persons-email')
+def _get_placeholders(sender, person, event, **kwargs):
```

Comment:
`def _get_placeholders(sender, person, event, register_link=False, **kwargs):` (and adapt the check below etc accordingly)

We only accept `**kwargs` in signals so they don't break if someone adds extra arguments when calling them - but arguments that are actually used in a signal handler should always be explicitly specified.